meta-resin-common/recipes-support/resin-mounts/resin-mounts-flasher/mnt-boot-config.json.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Flasher conf partition mountpoint
+Requires=mnt-boot.mount temp-conf.service tmp.mount
+After=mnt-boot.mount temp-conf.service tmp.mount
+Before=resin-device-uuid.service
+
+[Mount]
+What=/tmp/conf/config.json
+Where=/mnt/boot/config.json
```

Comment:
So this service depends on temp-conf.service. This temp-conf.service file will do:

ExecStart=@BASE_BINDIR@/cp -vf /mnt/boot/config.json /tmp/conf/config.json || @BASE_BINDIR@/touch /tmp/conf/config.json

Then, this mnt-boot-config.json.mount unit will mount back this copy to /mnt/boot/config.json shadowing the original file so the original is hidden remaining unchanged for other board provisionings and this mounted one can be appended the UUID. Is this correct?